com.unity.ml-agents/Documentation~/com.unity.ml-agents.md

```diff
@@ -58,6 +58,9 @@ This version of the Unity ML-Agents package is compatible with the following ver
 
 ## Known limitations
 
+### Training
```

Comment:
Do we need to say anything about Inference/Barracuda?

Reply:
They don't call out any platform limitations in their docs so i'm not sure what to point to.

Reply:
Sounds good.